tests/tests/test_monitor_client.py

```diff
@@ -470,6 +470,10 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
             user_name,
             "OK: Monitor Alert for Log file contains " + log_pattern + " on " + devid,
         )
+        # in each CRITICAL and OK email we expect the pattern to be present at least 3 times
+        assert mail.count(log_pattern) >= 6
+        # in each CRITICAL and OK email we expect the log path to be present at least 1 time
+        assert mail.count(log_pattern) >= 2
```

Comment:
present at least `2` times it looks like, no :thinking:

Reply:
> present at least `2` times it looks like, no 🤔

nice that you read the comments! it is present at least 2 times in the emails, but we have two emails. so it is 1 each.